source/_components/rainforest_eagle.markdown

```diff
@@ -0,0 +1,72 @@
+---
+layout: page
+title: "Rainforest Eagle-200"
+description: "Instructions on how to setup the Rainforest Eagle-200 with Home Assistant."
+date: 2019-07-02 18:00
+logo: rainforest_automation_logo.png
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category:
+  - Energy
+ha_release: 0.96
+ha_iot_class: Local Polling
+redirect_from:
+ - /components/sensor.rainforest_eagle/
+---
+
+A sensor platform for the [Rainforest Eagle-200](https://rainforestautomation.com/rfa-z114-eagle-200/) energy gateway.
+
+### {% linkable_title Configuration %}
+
+To enable this sensor, add the following lines to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: rainforest_eagle
+    ip_address: IP_FOR_EAGLE
+    cloud_id: CLOUD_ID_FROM_EAGLE
+    install_code: INSTALL_CODE_FROM_EAGLE
+
+```
+
+```yaml
+# example configuration.yaml entry, limiting the metrics to demand only
```

Comment:
Move this example underneed the configuration variables and give it a full example title

Reply:
Moved the example underneath the configuration variables and created an examples section with two examples.